File: InvoiceProject/invoice-server-starter/src/main/java/cz/itnetwork/entity/InvoiceStatistics.java
package cz.itnetwork.entity;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class InvoiceStatistics {

    private double currentYearSum;

    private double allTimeSum;

    private long invoicesCount;
}

File: InvoiceProject/invoice-client-starter/src/invoices/InvoiceStatistics.js
import React, {useState, useEffect} from "react";
import { apiGet } from "../utils/api";

const InvoiceStatistics = () => {

    const [statistics, setStatistics] = useState({});

    useEffect(() => {
        apiGet("/api/invoices/statistics").then((data) => setStatistics(data));
    }, [])

    return (
    <div>
        <table className="table d-flex flex-row gap-3">
            <tr>
                <th>Počet faktur:</th>
                <td>{statistics.invoicesCount}  </td>
            </tr>
            <tr>
                <th>Tržby tento rok:</th>
                <td>{statistics.currentYearSum},- Kč</td>
            </tr>
            <tr>
                <th>Celkové tržby:</th>
                <td>{statistics.allTimeSum},- Kč</td>
            </tr>
        </table>
    </div>
    );
}

export default InvoiceStatistics;